probes/notArchived/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: notArchived
+short: Check that the project is archvied
```

Comment:
the description says the opposite of the probeID, I think?

Reply:
> Should thee prove be called repositoryIsArchived instead?

Then we have the dilemma:

1. If we return `OutcomePositive`, we convey that it is positive that the project is archived.
2. If we return `OutcomeNegative`, the return value is negative, but we consider it positive, since it is positive that the repository is not archived.

Reply:
Since these probe names aren't exposed to users yet, and the problem is bigger than just the Maintained checks, I've opened #3654 for this issue. Resolving for now.